easybuild/easyconfigs/g/GST-plugins-base/GST-plugins-base-1.9.1-foss-2016a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'GST-plugins-base'
+version = '1.9.1'
+
+homepage = 'http://gstreamer.freedesktop.org/'
+description = """GStreamer is a library for constructing graphs of media-handling
+ components. The applications it supports range from simple
+ Ogg/Vorbis playback, audio/video streaming to complex audio
+ (mixing) and video (non-linear editing) processing."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://gstreamer.freedesktop.org/src/gst-plugins-base']
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+  ('GStreamer', '1.9.1'),
+  ('Automake', '1.15'),
+  ('Autoconf', '2.69'),
```

Comment:
@RvDijk `Automake` and `Autoconf` are just build dependencies, and it's better to use the `Autotools` bundle we have (which also included `libtool`)